chevah/compat/tests/elevated/test_capabilities.py

```diff
@@ -77,11 +77,30 @@ def test_getCurrentPrivilegesDescription(self):
                 'SeIncreaseWorkingSetPrivilege:0, SeTimeZonePrivilege:0, '
                 'SeCreateSymbolicLinkPrivilege:0'
                 )
-            if self.os_version == 'nt-10.0':
-                # On Win 2016 we have an extra capability by default.
+            if int(self.os_version.split('.')[2]) >= 14393:
+                # Starting with Win 2016 there's an extra default capability.
                 expected_capabilities += (
                     ', SeDelegateSessionUserImpersonatePrivilege:0'
                     )
+            if int(self.os_version.split('.')[2]) >= 17763:
```

Comment:
I think that we need a special os_build variable.... it doesn't make sense to  construct the os_version with the build number just to deconstruct it each time we need to get the build number.

the above condition should be just

```
if self.os_build >= 14393:
    # Do something
````